src/main.py

```diff
@@ -29,7 +30,7 @@ def signal_handler(sig, frame):
 
     shutdown_commands = ShutdownCommands()
 
-    server = CommunicationServer(shutdown_commands, host=SHUTDOWN_EVENTS_HOST, port=SHUTDOWN_EVENTS_PORT)
+    server = CommunicationServer(shutdown_commans, host=SHUTDOWN_HOST, port=SHUTDOWN_PORT)
```

Comment:
```suggestion
    server = CommunicationServer(shutdown_commands, host=SHUTDOWN_HOST, port=SHUTDOWN_PORT)
```

@aitorcas23, se acaba de colar de nuevo el "commans", aunque esta vez la definición de la variable es correcta 😜